Sources/ParseLiveQuery/Client.swift

```diff
@@ -149,13 +150,18 @@ extension Client {
             )
             subscriptions.append(subscriptionRecord)
 
-            if socket == nil {
-                if !disconnected {
-                    reconnect()
-                }
-            } else {
-                sendOperationAsync(.Subscribe(requestId: subscriptionRecord.requestId, query: query))
+        if socket == nil || socket?.readyState != .OPEN{
+            
+            if !userDisconnected{
+                reconnect()
+            }else{
+                print("Warning: The client was explicitly disconnected! You must explicitly call .reconnect() in order to process your subscriptions.")
             }
+            
+        } else if socket?.readyState == .OPEN{
```

Comment:
Nit: Space before `{`. I know this is tedious, but this is the overall code style that we are using.